common/lib/xmodule/xmodule/course_module.py

```diff
@@ -335,6 +335,15 @@ class CourseFields(object):
         help=_("Enter the unique identifier for your course's video files provided by edX."),
         scope=Scope.settings
     )
+    facebook_url = String(
+        help=_(
+            "Enter the URL for the official course Facebook group. "
+            "If you provide a URL, the edX mobile app includes a button that students can tap to access the group."
+        ),
```

Comment:
Here's the hipchat response from Lou on this (in January):
```
@LouWang For the new Studio Advanced Setting for the URL to the Official Course FB page, does it make sense to keep it specific to FB or generalize it as Social Group URL? 
10:54 AM Lou Wang i think it should be specific, since the app UI exposes it as a FB link
```